geetools/Image.py

```diff
@@ -1427,3 +1427,18 @@ def plot(
             gdf = gpd.GeoDataFrame.from_features(fc.getInfo()["features"])
             gdf = gdf.set_crs("EPSG:4326").to_crs(crs)
             gdf.boundary.plot(ax=ax, color=color)
+
+    @classmethod
+    def fromList(cls, image_list: ee_list):
```

Comment:
I recently dropped (https://github.com/gee-community/geetools/pull/346) the `ee_list` type as it was preventing intersphinx to work correctly. you can replace it by `ee.List | list` as we now dropped 3.8 and can fully used annotation. 

Also I mention in the documentation that `ee` objects are mentioned in parameters using camelCase so if you support both you need to play with the naming convention and use a single word ;-) what about `images` ?